distributed/scheduler.py

```diff
@@ -3860,8 +3861,13 @@ async def start_unsafe(self):
 
         for listener in self.listeners:
             logger.info("  Scheduler at: %25s", listener.contact_address)
-        for k, v in self.services.items():
-            logger.info("%11s at: %25s", k, "%s:%d" % (listen_ip, v.port))
+        for name, server in self.services.items():
+            if name == "dashboard":
+                addr = listener.contact_address.split("://")[1].split(":")[0]
```

Comment:
Minor comment -- we have a `get_address_host` utility (already imported in this module) that _I think_ does the same thing as the logic here